T/tblis/build_tarballs.jl

```diff
@@ -0,0 +1,44 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "tblis"
+version = v"1.2.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/devinamatthews/tblis.git", "3e4c4b82943726c443b6f408c9c9791dcad7a847")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd tblis/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+make -j${nproc}
+cp lib/.libs/libtblis.so.0.0.0 /workspace/destdir/lib/libtblis.so.0
+cp src/external/tci/lib/.libs/libtci.so.0.0.0 /workspace/destdir/lib/libtci.so.0
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:x86_64, libc=:glibc),
+    Linux(:x86_64, libc=:musl)
+]
```

Comment:
```
┌ Warning: /agent/_work/1/s/T/tblis/build/x86_64-linux-gnu/DMIpqpKO/destdir/lib/libtblis.so.0 contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
│ 
│     platforms = expand_cxxstring_abis(platforms)
└ @ BinaryBuilder /depot/packages/BinaryBuilder/W9GL3/src/auditor/compiler_abi.jl:222
##[warning]/agent/_work/1/s/T/tblis/build/x86_64-linux-gnu/DMIpqpKO/destdir/lib/libtblis.so.0 contains std::string values!  This causes incompatibilities across the GCC 4/5 version boundary.  To remedy this, you must build a tarball for both GCC 4 and GCC 5.  To do this, immediately after your `platforms` definition in your `build_tarballs.jl` file, add the line:
```
```suggestion
platforms = [
    Linux(:x86_64, libc=:glibc),
    Linux(:x86_64, libc=:musl)
]
platforms = expand_cxxstring_abis(platforms)
```
Also, any particular reason why you're building only for these two platforms?